source/features/easy-toggle-commit-messages.tsx

```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function toggleCommitMessage(event: MouseEvent): void {
+	const elementClicked = event.target as HTMLElement;
+	const commitBox = elementClicked.closest('.js-commits-list-item')!;
+
+	// The clicked element is not a button or a link
+	if (elementClicked === commitBox || elementClicked.matches('div, p, pre')) {
+		select('.ellipsis-expander', commitBox)!
+			.dispatchEvent(new MouseEvent('click', {bubbles: true, altKey: event.altKey}));
```

Comment:
Propagating `altKey` will be useful in conjunction with #5345

Reply:
Does this work?
```suggestion
			.click();
```